Assets/MRTK/Providers/Oculus/XRSDK/OculusXRSDKDeviceManager.cs

```diff
@@ -45,6 +45,13 @@ public class OculusXRSDKDeviceManager : XRSDKDeviceManager
         private OVRHand leftHand;
         private OVRMeshRenderer leftMeshRenderer;
         private OVRSkeleton leftSkeleton;
+
+
+        /// <summary>
+        /// The profile that contains settings for the Leap Motion Device Manager input data provider.  This profile is nested under 
+        /// Input > Input Data Providers > Leap Motion Device Manager in the MixedRealityToolkit object in the hierarchy.
+        /// </summary>
+        public OculusXRSDKDeviceManagerProfile settingsProfile => ConfigurationProfile as OculusXRSDKDeviceManagerProfile;
```

Comment:
Does this need to be `public`?

Reply:
Also, since it's a property, [should be `PascalCase`](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/CodingGuidelines.html#naming-conventions)

Reply:
oops  I reverted the pascal case when rebasing 🦖 
done!